### PR TITLE
mark web_advanced_search_x2x as installable

### PR DESCRIPTION
**Pull request by rami-wafaie**

### CODE REVIEW

File: web_advanced_search_x2x/__manifest__.py [hunk 1]
```diff
@@ -1,26 +1,10 @@
...
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © initOS GmbH 2017
```
Comment (hbrunn) [inline web_advanced_search_x2x/__manifest__.py:2]: overwriting someone else's copyright notice is an absolutely deadly sin

File: web_advanced_search_x2x/__init__.py [hunk 2]
```diff
@@ -1,20 +1,3 @@
...
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © initOS GmbH 2017
```
Comment (hbrunn) [inline web_advanced_search_x2x/__init__.py:2]: same here

**Review by hbrunn — CHANGES_REQUESTED** [review]
you just took the 8.0 version of the code, but what you need to do is to first import all changes from 9.0, then change what's necessary to change

File: web_advanced_search_x2x/__manifest__.py (hunk 1 above)
Comment (rami-wafaie) [reply web_advanced_search_x2x/__manifest__.py:2]: I did it fastly sorry I will close this MR and open a new one with the right changes

**Pull request closed without merging**